examples/create_topic.rs

```diff
@@ -0,0 +1,42 @@
+use anyhow::{Context, Result};
+use kafka_protocol::messages::create_topics_request::{CreatableTopicBuilder, CreateTopicsRequestBuilder};
+use kafka_protocol::messages::TopicName;
+use kafka_rs::{Client, IndexMap, StrBytes};
+use tracing_subscriber::prelude::*;
+
+#[tokio::main(flavor = "current_thread")]
+async fn main() -> Result<()> {
+    tracing_subscriber::registry()
+        // Filter spans based on the RUST_LOG env var.
+        .with(tracing_subscriber::EnvFilter::new("error,simple_producer=debug,kafka_rs=debug"))
+        .with(
+            tracing_subscriber::fmt::layer()
+                .with_target(false) // Too verbose, so disable target.
+                .with_level(true) // Shows tracing level: debug, info, error &c.
+                .compact(),
+        )
+        // Install this registry as the global tracing registry.
+        .try_init()
+        .expect("error initializing tracing");
+
+    let cli = Client::new(vec!["localhost:19092".into()]);
```

Comment:
@thedodd do you want me to use env variables for:
- Topic
- Host
- Port

I hardcoded this just based upon my own setup